src/main/cpp/t04_students.cpp

```diff
@@ -29,10 +29,37 @@
 
 #include "t04_students.h"
 #include <iostream>
-
+#include <set>
+#include <map>
 using namespace std;
 
 int t04_students() {
-    
+    int n;
+    string j;
+    float s=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 's' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4684787)